README.md

```diff
@@ -4,3 +4,67 @@ A library used in Open Permissions Platform services for accessing registry
 data stored in CouchDB.
 
 __NOTE: To use this library CouchDB must be configured to support Python views. See <https://pythonhosted.org/CouchDB/views.html>__
+
+
+## Migrations
+
+There is support for simple migrations when schemas change.
+
+To create an empty migration file using the command line interface, run:
```

Comment:
Its not an empty file is it? It looks like it creates a template file of some description.